Apply updates from notion to /pricing/ (AM3) (#13183)
getsentry/sentry-docs: docs/pricing/index.mdx

| >10M-20M     | $0.0001300    | $0.0001625 | $0.0002600        | $0.0003250    |
| >20M         | $0.0001200    | $0.0001500 | $0.0002400        | $0.0003000    |

#### Tracing Pricing

Tracing is enabled by and will be billed for in spans.

| Span Volume | Team Reserved | Team PAYG  | Business Reserved | Business PAYG |
| ----------- | ------------- | ---------- | ----------------- | ------------- |
| >10M-100M   | $0.0000016    | $0.0000020 | $0.0000032        | $0.0000040    |
| >100M       | $0.0000014    | $0.0000018 | $0.0000029        | $0.0000036    |

#### Replays Pricing

| Replay Volume | Team Reserved | Team PAYG  | Business Reserved | Business PAYG |
| ------------- | ------------- | ---------- | ----------------- | ------------- |
| >50-5k        | $0.0030000    | $0.0037500 | $0.0030000        | $0.0037500    |
| >5k-100k      | $0.0028500    | $0.0035630 | $0.0028500        | $0.0035630    |
| >100k-900k    | $0.0025650    | $0.0032060 | $0.0025650        | $0.0032060    |
| >900k-4.5M    | $0.0023090    | $0.0028860 | $0.0023090        | $0.0028860    |
| >4.5M         | $0.0019620    | $0.0024530 | $0.0019620        | $0.0024530    |

#### Profiling Pricing

> Profile Hours are determined by the length of the profile in milliseconds and are billed in hours. Partial hours are prorated and rounded down to the nearest cent. Profile Hours can't be purchased in advance and are only available through your PAYG budget.

Reviewer: More explicitly, does this mean that the only way to purchase more profile hours is via a PAYG budget? If so, I'd suggest changing this to: 

```suggestion
Profile Hours are determined by the length of the profile in milliseconds and are billed in hours. Partial hours are prorated and rounded down to the nearest cent. Profile Hours can only be purchased with your PAYG budget.
```

**Continuous Profile Hours**

| Team PAYG    | Business PAYG |
| ------------ | ------------- |
| $0.0315/hour | $0.0315/hour  |

**UI Profile Hours**

| Team PAYG  | Business PAYG |
| ---------- | ------------- |
| $0.25/hour | $0.25/hour    |

#### Monitors Pricing

All Sentry plans include one cron monitor and one uptime monitor. Additional monitors can't be purchased in advance and are only available through your PAYG budget.

<Alert>
  More details on how to activate, deactivate, and manage your monitors quota is
  available under: 
  - [Billing Quota Management: Manage Your Cron Monitors](/pricing/quotas/manage-cron-monitors/)
  {/* TODO: Link to the page below when it exists */}

>   - Billing Quota Management: Manage Your Uptime Monitors

Reviewer: It looks like this is missing a link

</Alert>

**Cron Monitors**

| Team PAYG     | Business PAYG |
| ------------- | ------------- |
| $0.78/monitor | $0.78/monitor |

**Uptime Monitors**

| Team PAYG     | Business PAYG |
| ------------- | ------------- |
| $1.00/monitor | $1.00/monitor |

#### Attachments Pricing (per GB)

| Attachment Size | Team Reserved | Team PAYG | Business Reserved | Business PAYG |
| --------------- | ------------- | --------- | ----------------- | ------------- |
| >1GB            | $0.2500       | $0.3125   | $0.2500           | $0.3125       |

## Billing Cycles

<Alert>
  Billing or Owner-level users have the permissions to make changes to

>   information on the Subscription page or access invoices. If you need an

Reviewer: ```suggestion
  information on the Subscription page and access invoices. If you need an
```

Users with [Billing or Owner-level](/organization/membership/) permissions can find their invoices by going to the [**Settings**](https://sentry.io/orgredirect/organizations/:orgslug/settings/) page and clicking “Subscription” (under the “Usage & Billing” heading.)

### Changing Plans Mid-cycle

If you decide to increase your PAYG budget in the middle of your billing cycle, Sentry will start processing additional data as soon as the increase becomes effective.\*

If you want to decrease your PAYG budget in the middle of your billing cycle, but have already consumed more data than the new desired amount allows, your budget will be lowered to match what you've already used. Any new data will be rejected. Your PAYG budget will be decreased to the desired amount on your next billing date.

<Alert>
  If you upgrade from a Team to a Business plan mid-cycle, any existing PAYG
  budget will be recalculated based on Business tier pricing. This may mean that
  your PAYG budget will be consumed faster than it was before.
</Alert>

> \*In most cases, new budgets will take effect within minutes. Otherwise, we guarantee a maximum turnaround time of 24 hours.

Reviewer: This asterisk looks a little odd after an alert. I wonder if italicizing the line would look better, and still grab attention?
